changelog.md

```diff
@@ -1,11 +1,35 @@
 
+
+## 3.2.1 / 2020-11-04
+
+### New Features
+
+- Full implementation of spaces access types: open, on request and confidential (spaces available in Tracim < 3.2.1 are considering like type confidential)
+- Allow user to discover and join spaces
+- New plugins available:
+  - Adds every new user to all open spaces and to every newly created open space. You need to activate this plugin `tracim_backend_autoinvite` (documentation avialable [here](backend/official_plugins/README.md))
+  - Recursively adds new members of space to its parents with the default user role of each space. You need to activate this plugin `tracim_backend_parent_access` (documentation avialable [here](backend/official_plugins/README.md))
+  - When a user is removed from a space, this plugin recursively removes this user from the children of this space. You need to activate this plugin `tracim_backend_child_removal` (documentation avialable [here](backend/official_plugins/README.md))
+- Group vendor dependencies to make faster and smaller frontend builds (feature for developers and production)
```

Comment:
It also makes Tracim way more lightweight, enabling usage of Tracim on cheap / older device such as my cheap old phone and reducing bandwidth consumption when loading tracim (making Tracim faster to load on small connections)

Reply:
```suggestion
- vendor dependencies are grouped, making Tracim faster to load on small connections and older devices, and more lightweight on memory consumption. For developers and sys admins, this also makes Tracim build faster.
```